app/routers/profile.py

```diff
@@ -23,18 +24,31 @@
 
 def get_placeholder_user():
     return User(
+        id=1,
         username='new_user',
         email='my@email.po',
         password='1a2s3d4f5g6',
         full_name='My Name'
     )
 
 
+def get_placeholder_event():
+    return Event(
+        id=1,
+        title="new_event",
+        content="content_of_the_event",
+        start=datetime.date(2007, 5, 26),
+        end=datetime.date(2007, 5, 27),
+        owner_id=1
+    )
+
+
 @router.get("/")
 async def profile(
         request: Request,
         session=Depends(get_db),
-        new_user=Depends(get_placeholder_user)):
+        new_user=Depends(get_placeholder_user),
+        new_event=Depends(get_placeholder_event)):
 
     # Get relevant data from database
     upcouming_events = range(5)
```

Comment:
Maybe instaed of hard-coded "5", make it configurable?

Reply:
it is not my line, once the user will able to create events, probably this line will be deleted